app/templates/buyers/brief_publish_confirmation.html

```diff
@@ -34,20 +34,39 @@
+
     {% if not published %}
-      <p class="padding-bottom-small">All requirements are published on the Digital Marketplace where anyone can see them.</p>
-      <p class="padding-bottom-small">All requirements are open for {{ dates.application_open_weeks }} weeks.</p>
-      <p class="padding-bottom-small">If you publish your requirements today ({{ dates.today | shortdateformat }}), suppliers will be able to apply until {{ dates.closing_time }}, {{ dates.closing_date | shortdateformat}}.</p>
+      <div class="dmspeak">
+        <p>All requirements are published on the Digital Marketplace where anyone can see them.</p>
+      </div>
+        {% if brief.lotSlug == 'digital-specialists' and not brief.requirementsLength %}
+          <p><a href="{{ url_for('.edit_brief_question', framework_slug=brief.frameworkSlug, lot_slug=brief.lotSlug, brief_id=brief.id, section_slug='set-how-long-your-requirements-will-be-live-for', question_id='requirementsLength') }}">Set how long your requirements will be open for</a></p>
+          <div class="dmspeak">
+            <p>This will show you what the supplier application deadline will be if you publish your requirements today.</p>
+          </div>
+        {% else %}
+          <p>Your requirements will be open for {{ dates.application_open_weeks }}.</p>
+          <div class="dmspeak">
+            <p>If you publish your requirements today ({{ dates.published_date | shortdateformat }}), suppliers will be able to apply until {{ dates.closing_time }}, {{ dates.closing_date | shortdateformat}}.</p>
+          </div>
+        {% endif %}
     {% endif %}
-    <p><strong>Supplier questions will be sent to:</strong></p>
-    <p class="padding-bottom-small">{{ email_address }}</p>
-    <p class="padding-bottom-small">Ensure this email address {% if not published %}will be{% else %}is{% endif %} monitored. If you’re away while suppliers can still ask questions, you should make sure your emails are forwarded to a colleague.</p>
+
+    <p>Supplier questions will be sent to:</p>
+    <div class="dmspeak">
+      <p>{{ email_address }}</p>
+    </div>
+    <div class="dmspeak">
+      <p>Make sure this email address {% if not published %}will be{% else %}is{% endif %} monitored. If you’re away while suppliers can still ask questions, you should make sure your emails are forwarded to a colleague.</p>
+    </div>
 
     {% if not published and unanswered_required > 0 %}
-      <p class="padding-bottom-small"><strong>You still need to complete the following questions before your requirements can be published:</strong></p>
+      <div class="dmspeak">
+        <p><strong>You still need to complete the following questions before your requirements can be published:</strong></p>
+      </div>
       {% for section in sections %}
         {% for question in section.questions %}
-          {% if question.answer_required %}
+          {% if question.answer_required and not question.id == 'requirementsLength' %}
```

Comment:
Just confirming that this is what we want.

### Requirements Length + 1 other question to answer
![screen shot 2016-07-29 at 08 45 51](https://cloud.githubusercontent.com/assets/2454380/17241610/943874e4-556a-11e6-9986-b282e8bb74a5.png)

### Requirements length is the only question left to answer
![screen shot 2016-07-29 at 08 46 20](https://cloud.githubusercontent.com/assets/2454380/17241625/a50f290c-556a-11e6-92d2-8d1fe69a232f.png)

***

We think this looks dumb, right?

### Requirements length is the only question left to answer
![screen shot 2016-07-29 at 08 53 49](https://cloud.githubusercontent.com/assets/2454380/17241641/ca6e7d74-556a-11e6-8d38-80a7fb81e748.png)